com.ibm.wala.cast/source/java/com/ibm/wala/cast/ir/ssa/SSAConversion.java

```diff
@@ -740,12 +740,8 @@ protected boolean skip(int i) {
       ssa.perform();
       if (DEBUG_UNDO) System.err.println(("<<< done " + ir.getMethod()));
       return ssa.getComputedLocalMap();
-    } catch (RuntimeException e) {
-      //      System.err.println(("exception " + e + " while converting:"));
-      //      System.err.println(ir);
-      throw e;
-    } catch (Error e) {
-      //      System.err.println(("error " + e + " while converting:"));
+    } catch (RuntimeException | Error e) {
```

Comment:
Can't we just delete this try-catch?  It's just re-throwing

Reply:
Presumably this clause used to be considered useful when it printed an error message, which is now commented out. I will remove the entire `try`–`catch` per your suggestion.